RecoTracker/MkFit/plugins/createCMS2017.cc

```diff
@@ -0,0 +1,372 @@
+//-------------------
+// CMS 2017 geometry
```

Comment:
That's just the Tracker geometry right? why CMS?
Also how specific is this for 2017? Maybe `Phase1TrackerGeometry` is more appropriate here.

Reply:
Thanks for the suggestion, will rename (these were remnants of mkfit-internal names).